resources/europe/catalan_countries/catalan-mastodon.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "cat-mastodon",
+  "type": "mastodon",
+  "account": "osmcatala",
+  "locationSet": {"include": ["catalan_countries.geojson"]},
+  "languageCodes": ["ca"],
+  "order": 8,
```

Comment:
For reference, this will appear next to Matrix and Telegram – below Discourse, talk-cat, and the wiki but above Facebook, Twitter, GitHub, and YouTube.